ui/components/multichain/notifications-tag-counter/notifications-tag-counter.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  selectIsMetamaskNotificationsEnabled,
+  selectIsSnapNotificationsEnabled,
+  selectIsFeatureAnnouncementsEnabled,
+  getFeatureAnnouncementsUnreadCount,
+  getOnChainMetamaskNotificationsUnreadCount,
+} from '../../../selectors/metamask-notifications/metamask-notifications';
+import { Box, IconName, Icon, Text } from '../../component-library';
+import { getUnreadNotificationsCount } from '../../../selectors';
+import {
+  BackgroundColor,
+  BorderColor,
+  BorderRadius,
+  BorderStyle,
+  Display,
+  TextColor,
+  TextVariant,
+  IconColor,
+} from '../../../helpers/constants/design-system';
+
+type NotificationsTagCounterProps = {
+  noLabel?: boolean;
+};
+
+export const NotificationsTagCounter = ({
+  noLabel = false,
+}: NotificationsTagCounterProps) => {
+  const unreadNotificationsCount = useSelector(getUnreadNotificationsCount);
+  const isMetamaskNotificationsEnabled = useSelector(
+    selectIsMetamaskNotificationsEnabled,
+  );
+  const isSnapNotificationsEnabled = useSelector(
+    selectIsSnapNotificationsEnabled,
+  );
+  const isFeatureAnnouncementsEnabled = useSelector(
+    selectIsFeatureAnnouncementsEnabled,
+  );
+  const featureAnnouncementsUnreadCount = useSelector(
+    getFeatureAnnouncementsUnreadCount,
+  );
+  const onChainMetamaskNotificationsUnreadCount = useSelector(
+    getOnChainMetamaskNotificationsUnreadCount,
+  );
```

Comment:
We will need to ensure that these counters are valid when a user turns off/on the different notifications.

If this is a bug, lets either fix this or remove the advanced notification settings panel (that has been causing us headaches and may need rethinking)

Reply:
Edge-case, will create a ticket for this.

Reply:
NVM, look below we already add the counter correctly 👍🏾